app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if (command[0] == "cp"
+            and len(command) == 3
+            and command[1] != command[2]):
```

Comment:
The condition `command[1] != command[2]` is correctly checking if the source and destination filenames are different. However, you should also handle the case where they are the same by doing nothing, as per the task requirements.